runtime/browser/lib/demo-base.js

```diff
@@ -37,33 +37,18 @@ class DemoBase extends HTMLElement {
     this._arc = arc;
     this.update();
   }
-  get stages() {
-    return this._stages;
-  }
-  set stages(stages) {
-    this.stageNo = 0;
-    this._stages = stages;
-    this.update();
-  }
   update() {
-    if (this.arc && this.stages) {
+    if (this.arc) {
       this.nextStage();
```

Comment:
do we still need stageNo? maybe you could just call this.suggest() here and delete the nextStage() method?